syntax = "proto2";

package MMPro.micromsg;

import "BaseResponse.proto";
import "ChatRoomMemberData.proto";

message GetChatroomMemberDetailResponse {
	required BaseResponse _BaseResponse = 1;
	optional string _ChatroomUserName = 2;
	required uint32 _ServerVersion = 3;
	required ChatRoomMemberData _NewChatroomData = 4;
}
